clients/drcachesim/tools/view.cpp

```diff
@@ -534,6 +534,15 @@ view_t::parallel_shard_memref(void *shard_data, const memref_t &memref)
             disasm.insert(newline, " (taken)");
         else if (memref.instr.type == TRACE_TYPE_INSTR_UNTAKEN_JUMP)
             disasm.insert(newline, " (untaken)");
+        if (trace_version_ >= TRACE_ENTRY_VERSION_BRANCH_INFO &&
+            (memref.instr.type == TRACE_TYPE_INSTR_INDIRECT_JUMP ||
+             memref.instr.type == TRACE_TYPE_INSTR_INDIRECT_CALL ||
```

Comment:
Would be useful to add a utility func (if one doesn't exist already) to identify these three types.

Reply:
Oh right I was using `type_is_instr_branch() && !type_is_instr_direct_branch()` elsewhere.  I put that in here too.